src/clients/inst_autoconfigure.rb

```diff
@@ -39,7 +39,8 @@ def main
         "Profile general,mode:%1",
         Ops.get_map(Profile.current, ["general", "mode"], {})
       )
-      @need_systemd_isolate = true
+      @need_systemd_isolate = Ops.get_boolean(
+        Profile.current,["general", "mode", "activate_systemd_default_target"], true)
```

Comment:
Mmm, on the other hand, in this case ```Ops.get_boolean``` is actually more handy than the pure Ruby alternative, so never mind. I said nothing